Skip projects where toplevel build.gradle contains subprojects

timtebeek opened the pull request:
Alternatively we could recognize and skip such projects in the CLI when ingesting, until we support such a structure there.

knutwannheden reviewed (approved):
LGTM. I suppose we will have to use the GraphQL API to manually delete the existing LSTs for these two projects.

timtebeek commented:
Removed the Jenkins ingestion jobs just now.
```
Deleted Netflix/Raigad at line 7415 because it is marked as skipped: Toplevel build.gradle contains subprojects
Deleted Netflix/dyno at line 7432 because it is marked as skipped: Toplevel build.gradle contains subprojects
```

timtebeek commented:
Called remove on dyno and Raigad in app.moderne.io
![image](https://github.com/moderneinc/jenkins-ingest/assets/1027334/4e7197d5-3fb5-4b72-95a6-8339e7aff213)
